matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/MatrixConfiguration.kt

```diff
@@ -65,6 +65,12 @@ data class MatrixConfiguration(
          * Thread messages default enable/disabled value
          */
         val threadMessagesEnabledDefault: Boolean = false,
+
+        /**
+         * Sync loop parameter: The maximum time to wait, in milliseconds, before returning this request.
```

Comment:
This is the server which will wait. Maybe rephrase to make this clearer.

Reply:
updated